Scripts/Audio.cs

```diff
@@ -183,4 +183,138 @@ public string output
     }
 
 
+
+    [Serializable]
+    public class SpeechRecognitionOptions
```

Comment:
maybe put all the speech recognition stuff into a separate file to avoid cluttering `Audio`